src/plugins/workspace/server/plugin.ts

```diff
@@ -259,6 +264,7 @@ export class WorkspacePlugin implements Plugin<WorkspacePluginSetup, WorkspacePl
       maxImportExportSize,
       permissionControlClient: this.permissionControl,
       isPermissionControlEnabled,
+      identitySourceRegistry,
```

Comment:
How can we override the `identitySourceRegistry` implementation in the routes? I'm prefer update `identitySourceRegistry` to `getIdentitySourceRegistry`, then other plugins can call workspace plugin client to override the identity source registry outside.